share/man/man7/hier.7

```diff
@@ -462,161 +826,277 @@ The layout of the source tree is described by the top-level
 file.
 .Pp
 .It Pa tests/
-the
+The
 .Fx
 test suite; see
 .Xr tests 7
 .El
 .It Pa /var/
-log, temporary, transient, and spool files
+Log, temporary, transient, and spool files:
 .Pp
-.Bl -tag -width "preserve/" -compact
+.Bl -tag -width "defaults/" -compact
 .It Pa account/
-system accounting files
+System accounting files:
 .Pp
-.Bl -tag -width Fl -compact
+.Bl -tag -width Ds -compact
 .It Pa acct
-execution accounting file; see
-.Xr acct 5
+Execution accounting file;
+see
+.Xr acct 5 .
 .El
 .Pp
 .It Pa at/
-timed command scheduling files; see
-.Xr at 1
+Timed command scheduling files;
+see
+.Xr at 1 :
 .Pp
-.Bl -tag -width Fl -compact
+.Bl -tag -width Ds -compact
 .It Pa jobs/
-job files
+Job files.
 .It Pa spool/
-output spool files
+Output spool files.
 .El
 .Pp
+.It Pa audit/
+Security event audit trail files; see
+.Xr audit 8 .
+.It Pa authpf/
+User shell sessions for authenticating gateways; see
+.Xr authpf 8 .
 .It Pa backups/
-miscellaneous backup files
+Critical system configuration backups.
 .It Pa cache/
-miscellaneous cache files
+Miscellaneous cached files:
 .Pp
-.Bl -tag -width Fl -compact
+.Bl -tag -width Ds -compact
 .It Pa pkg/
-cached packages for
-.Xr pkg 8
+Cached packages for
+.Xr pkg 8 .
+.It Pa cups/
+Cached printers for the Common Unix Prinitng system; see
+.Xr cups 1 .
 .El
 .Pp
 .It Pa crash/
-default directory for kernel crash dumps; see
+Default directory to store kernel crash dumps; see
 .Xr crash 8
 and
-.Xr savecore 8
+.Xr savecore 8 .
 .It Pa cron/
-.Xr cron 8
-files
+Files used by cron;
+see
+.Xr cron 8 :
 .Pp
-.Bl -tag -width Fl -compact
+.Bl -tag -width Ds -compact
 .It Pa tabs/
-.Xr crontab 5
-files
+Crontab files;
+see
+.Xr crontab 5 .
 .El
 .Pp
 .It Pa db/
-miscellaneous automatically-generated system-specific database files
+Miscellaneous automatically generated system-specific database files.
 .Pp
-.Bl -tag -width "freebsd-update/" -compact
+.Bl -tag -width 15n -compact
+.It Pa etcupdate/
+Temporary files and log for
+.Xr etcupdate 8 .
 .It Pa freebsd-update/
-temporary files and downloads for
-.Xr freebsd-update 8
+Downloads and temporary files for
+.Xr freebsd-update 8 .
+.It Pa pkg/
+Package database.
 .El
 .Pp
 .It Pa empty/
-for use by programs that require an empty directory.
-Uses include privilege separation by
+Empty directory for use by programs that need a specifically empty directory.
+Used for instance by
 .Xr sshd 8
+for privilege separation.
 .It Pa games/
-miscellaneous game status and score files
+Miscellaneous game status and score files for
+.Ux
+heritage games.
 .It Pa heimdal/
 Kerberos server databases; see
-.Xr kdc 8
+.Xr kdc 8 .
+.It Pa lib/
+Holds state information for applications ported from linux.
 .It Pa log/
-miscellaneous system log files
+System log files:
 .Pp
-.Bl -tag -width "utx.lastlogin" -compact
+.Bl -tag -width 14n -compact
+.It Pa Xorg.0.log
```

Comment:
I am not sure what to think of this yet...

This page documents directories, not files. Yet I do understand the utility of adding it here...

Reply:
The convention is to document directories and not files, but afaik that isn't an explicit rule. I interpreted the term hierarchy as neutrally inclusive to either. I think the ratio of utility to maintenence burden of filling out /var/log is actually quite impressive.